README.md

```diff
@@ -118,6 +118,45 @@ def nextToken(self):
 NL: ('\r'? '\n' ' '*); #For tabs just switch out ' '* with '\t'*
 ```
 
+Usage (CSharp)
+=====
+
+![Build/Tests](https://github.com/yshavit/antlr-denter/workflows/CSharp/badge.svg)
+
+NuGet
+-----
+> TODO
+
+In Antlr:
+```antlrv4
+tokens { INDENT, DEDENT }
+
+@lexer::header {
+using AntlrDenter.DenterHelper;
+}
+
+@lexer::members {
+private DenterHelper denter;
+  
+public override IToken NextToken()
+{
+    if (denter == null)
+    {
+        denter = DenterHelper.Builder()
+            .Nl(NL)
+            .Indent(MyCoolParser.INDENT)
+            .Dedent(MyCoolParser.DEDENT)
+            .PullToken(base.NextToken);
+    }
+
+    return denter
+        .NextToken();
+}
```

Comment:
minor/formatting: this brace should be indented by one

Reply:
I found that the generated code was indented too far if indented here, not sure what to do about that

Reply:
Oh, interesting. Does that cause errors in C# (with standard linters, etc)? If so, we should write a comment so people don't get thrown off -- if not, this is fine.